puppet/modules/jenkins_job_builder/files/centos.org/ansible/jenkins_job.yml

```
@@ -101,6 +103,20 @@
       retries: 360
       delay: 30
 
+    - name: "Create artifacts folder"
```

Nit: directory is the unixy term

```suggestion
    - name: "Create artifacts directory"
```